packaging/helm/skipper-aws/templates/kube-aws-iam-controller/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+{{ if .Values.install_kube_aws_iam_controller }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kube-aws-iam-controller
+  namespace: kube-system
+  labels:
+    application: kube-aws-iam-controller
+    version: v0.1.0
```

Comment:
Would be good to pull this out into the values file.

Reply:
true all image versions should be there

Reply:
fixed